<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindow</class>
 <widget class="QDialog" name="AddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodawanie wyniku meczu</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>361</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="AddButton">
      <property name="text">
       <string>Dodaj mecz</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CancelButton">
      <property name="text">
       <string>Powrót</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="HomeTeamBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="AwayTeamBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gość:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gospodarz:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>56</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wynik:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="HomeScoreBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="AwayScorebox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>56</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Gość</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Gospodarz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
